docs/api-usage.md

```diff
@@ -129,6 +129,54 @@ curl --location --request POST 'https://api.postman.gov.sg/v1/campaign/100/uploa
 }'
 ```
 
+#### 3.4 Check processing status of uploaded file
+
+```bash
+curl --location --request GET 'https://api.postman.gov.sg/v1/campaign/100/upload/status' \
+--header 'Authorization: Bearer your_api_key'
+```
+
+While the csv is being processed, `is_csv_processing` would be `true`. When `is_csv_processing` is`false`, this indicates that the csv processing is complete. You can poll the `/upload/status` endpoint to refresh the file processing status.
```

Comment:
```suggestion
While the csv is being processed, `is_csv_processing` would be `true`. When `is_csv_processing` is`false`, this indicates that the csv processing is complete or an error has occurred. You can poll the `/upload/status` endpoint to refresh the file processing status.
```

Would be good to make it explicit that when `is_csv_processing` is `true`, it does not mean that the processing has succeeded. One of our API user had this misconception and was only checking for whether this value was `true`.